Dockerfile

```
@@ -0,0 +1,32 @@
+FROM alpine:3.15.0
+
+COPY ./docker_configs/entrypoint.sh /bin/entrypoint.sh
+COPY ./clair-scanner /clair-scanner
+
+RUN apk update && apk add \
+    curl \
+    zip \
+    alpine-sdk \
+    ca-certificates \
+    git\
+    mercurial \
+    breezy \
+    go \
+    && rm -rf /var/cache/apk/*
```

```suggestion
RUN apk update &&\
    apk add --no-cache \
    curl \
    zip \
    alpine-sdk \
    ca-certificates \
    git\
    mercurial \
    breezy \
    go
```

Que penses-tu d'utiliser le flag `--no-cache` au lieu de faire le rm à la mano?